element_calcium_imaging/scan.py

```diff
@@ -334,19 +336,13 @@ def make(self, key):
 
             # Insert in ScanInfo
             x_zero = (
-                scan.motor_position_at_zero[0]
-                if scan.motor_position_at_zero
-                else None
+                scan.motor_position_at_zero[0] if scan.motor_position_at_zero else None
             )
             y_zero = (
-                scan.motor_position_at_zero[1]
-                if scan.motor_position_at_zero
-                else None
+                scan.motor_position_at_zero[1] if scan.motor_position_at_zero else None
             )
             z_zero = (
-                scan.motor_position_at_zero[2]
-                if scan.motor_position_at_zero
```

Comment:
This code redundantly tests `scan.motoro_position_at_zero` three times.  How about:
```python
x_zero, y_zero, z_zero = scan_motor_position_at_zero or (None, None, None)
```

Reply:
Makes sense.